### DISCUSSION TITLE
Add /import switch to ep_setup

### DISCUSSION THREAD

**Discussion by kazigk** [valinet/ExplorerPatcher#2141] [Ideas]
Currently, there is no automated way to import the settings, and just importing the registry file is [not enough](https://github.com/valinet/ExplorerPatcher/wiki/Settings-management#export-the-current-settings) in some cases.

When it gets implemented, users could easily automate the setup of ExplorerPatcher with a simple winget command:
`winget install valinet.ExplorerPatcher --override "/import ExplorerPatcher_22621.1413.54.5.reg"`

Alternatively, this could be a part of a separate binary, or ExplorerPatcher could just look for a .reg file in a specific location on startup.

**Comment by pyrates999** [comment]
This should be moved to https://github.com/valinet/ExplorerPatcher/discussions/categories/ideas

**Comment by kazigk** [comment]
> This should be moved to https://github.com/valinet/ExplorerPatcher/discussions/categories/ideas

I thought we are supposed to [use issues](https://github.com/valinet/ExplorerPatcher/wiki/Reporting-problems) instead.

If I have a choice between discussions and issues, I'll always choose issues, because they can be easily mentioned in commit messages, pull requests, other issues, and offer better tracking in general. Currently, there are no clear guidelines.

**Comment by pyrates999** [comment]
To me, issues are for bugs.  New features go into ideas.  If I'm wrong, please correct me.
